utils.sh

```diff
@@ -411,6 +411,13 @@ function write_pull_secret() {
         return
     fi
 
+    if [ "${OPENSHIFT_CI}" == true ]; then
+        # Assume that the personal pull secret file we were given
+        # contains everything we need in CI.
+        cp "${PERSONAL_PULL_SECRET}" "${PULL_SECRET_FILE}"
```

Comment:
Yes, that's the plan. See https://github.com/dhellmann/openshift-release/commit/69603e19d6a68fed4b16e62e010fc1cd382acde8 for the WIP patch. I will propose that as a PR after this change merges (because if I do it before it merges, then the rehearse job will fail).